src/apps/webhooks.clj

```diff
@@ -58,3 +63,6 @@
 
 (defn list-topics []
   {:topics (select [:webhooks_topic :wt])})
+
+(defn list-types []
+  {:webhooktypes (select [:webhooks_type :wtypes])})
```

Comment:
There's no need to change this, but a table alias isn't really needed in cases where you're not doing a join.

Reply:
👍